flixel/addons/api/FlxGameJolt.hx

```diff
@@ -429,7 +429,7 @@ class FlxGameJolt
 	 * Retrieve the high scores from this game's remote data. If not authenticated, leaving Limit null will still return the top ten scores. Requires initialization.
 	 *
 	 * @see		https://gamejolt.com/api/doc/game/scores/fetch/
-	 * @param	TableID		The table you want to pull from
+	 * @param 	TableID		Optional: the ID of the table you want to pull data from. If null, the pulled score will be returned as the highes. Ignored if zero.
```

Comment:
The ID of the table you want to pull data from. If left blank, the scores from the primary score table will be returned.

Reply:
I just saw this thanks